test/user.test.js

```diff
@@ -294,17 +293,15 @@ describe('User', function() {
         },
         function(next) {
           User.login({email: 'b@c.com', password: 'bar'}, function(err, accessToken) {
-            accessTokenId = accessToken.userId;
             if (err) return next(err);
-            assert(accessTokenId);
+            assert(accessToken.userId);
```

Comment:
i was thinking the same thing when reviewing this file, it's a bit light as is. ok, will do where it makes sense